marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/FlowController.java

```diff
@@ -187,6 +188,31 @@ private RunFlowResponse runStepsInAFlow(String flowName, MultipartFile[] uploade
         return runFlowResponse;
     }
 
+    private String[] getStepNumbers(String flowName, String... stepNames) {
```

Comment:
Method signature can be String[] stepNames. No need to use a different format.

Reply:
It's more of a coding style as both serve the same purpose. But I changed it to String[] as it is more readable and also restricts the function to an array.